### PR TITLE
Update RHEL7-based images to Maven 3.6

### PR DESCRIPTION
**Pull request by jmtd**

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -42,13 +42,16 @@ modules:
   - name: cct_module
     git:
       url: https://github.com/jboss-openshift/cct_module.git
-      ref: 0.38.0
+      ref: master
```
Comment (jerboaa) [inline image.yaml]: I've always got a bad feeling when building from a changing target. Would we be able to add a tag to cct_module including the maven36 modules and use that instead? I'm thinking reproducible builds (at least in theory). Thoughts?

**Review by jerboaa — COMMENTED** [review]
I'd prefer if we used a tag in cct_modules.

File: image.yaml (hunk 1 above)
Comment (jmtd) [reply image.yaml]: I know what you mean. I hadn't cut a new tag in cct_modules at the time. I wonder if there's a risk that the union of all consumers of cct_modules might want tags often enough to clutter the tag namespace. On the other hand we can use git commit sha1 IDs as the ref value now (cf https://github.com/cekit/cekit/issues/618). What do you think about that?

Comment (jerboaa) [reply image.yaml]: That works for me. It at least fixes it to a specific revision.

**Review by jerboaa — APPROVED** [review]
Looks good.

**Pull request merged**